chaosaws/awslambda/actions.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+
+from chaosaws import aws_client
+from chaosaws.types import AWSResponse
+
+__all__ = ["put_function_concurrency", "delete_function_concurrency"]
+
+
+def put_function_concurrency(function_name: str,
+                             concurrent_executions: int,
+                             configuration: Configuration = None,
+                             secrets: Secrets = None) -> AWSResponse:
+    """
+    Throttles Lambda by setting reserved concurrency amount.
+    """
+    client = aws_client("lambda", configuration, secrets)
```

Comment:
wouldn't it be good to check that `function_name` and `concurrent_executions` are passed as parameters and not null at least?
I understand that `try` below is kind of guarding that, but I am afraid the error message could be sometimes misleading if you don't pass parameters

Reply:
Wouldn't you get a TypeError if they aren't passed in as parameters?